snap/snapcraft.yaml

```diff
@@ -152,32 +152,19 @@ parts:
   postgres-debs:
     plugin: nil
     stage-packages:
-      - perl
-      - perl-base
       - postgresql
       - util-linux
       - pgbouncer
       - pgbackrest
       - prometheus-postgres-exporter
-      - python3-psycopg2
       - python3-pysyncobj
       - python3-boto3
-      - postgresql-common
-      - postgresql-client-14
+      - patroni
+      # Landscape deps
       - postgresql-contrib
       - postgresql-14-debversion
       - postgresql-plpython3-14
       - python-apt-dev
```

Comment:
We seem to need those for integration with Landscape. Split them away, so we know why they are there.

Reply:
I'm not sure of the specifics, but those used to be installed via `extra-packages` config options before we switched to the snap and the landscape fail without them installed.